content/en/docs/ops/integrations/spire/index.md

```diff
@@ -226,7 +226,7 @@ To improve workload attestation security robustness, SPIRE is able to verify aga
 1. Deploy an example workload:
 
     {{< text bash >}}
-    $ istioctl kube-inject --filename @samples/security/spire/sleep-spire.yaml | kubectl apply -f -
+    $ istioctl kube-inject --filename samples/security/spire/sleep-spire.yaml | kubectl apply -f -
```

Comment:
Have you tried `@samples/security/spire/sleep-spire.yaml@` ?
See: https://github.com/istio/istio.io/blob/master/content/en/docs/setup/additional-setup/sidecar-injection/index.md

Reply:
`@samples/security/spire/sleep-spire.yaml@` will make it a clickable link in the rendered doc that will point to the file in the istio/istio repo (https://github.com/istio/istio/blob/master/samples/security/spire/spire-quickstart.yaml). That would be a good feature to use here, as @rvennam suggested. The two `@` chars are stripped when you copy/paste the command.

Reply:
Close the issue #11374 and keep the source document？

Reply:
@yanrongshi No, the fix for issue #11374 is to add a second `@` char at the end, not remove the first one.
```suggestion
    $ istioctl kube-inject --filename @samples/security/spire/sleep-spire.yaml@ | kubectl apply -f -
```